cpp/src/IceGrid/InternalRegistryI.cpp

```diff
@@ -185,7 +185,7 @@ NodePrxSeq
 InternalRegistryI::getNodes(const Ice::Current&) const
 {
     NodePrxSeq nodes;
-    Ice::ObjectProxySeq proxies = _database->getInternalObjectsByType(Node::ice_staticId());
+    Ice::ObjectProxySeq proxies = _database->getInternalObjectsByType(string{Node::ice_staticId()});
```

Comment:
We could rework the internal IceGrid API to avoid this allocation. For another PR.